core/source/File.mint

```diff
@@ -200,12 +185,11 @@ module File {
       input.multiple = true
       input.type = 'file'
 
-      document.body.appendChild(input)
```

Comment:
@gdotdesign Was that removed intentionally?
I'm asking since there's still `document.body.removeChild(input)` left beneath 🤔